content/05.main-text.md

```diff
@@ -56,7 +56,27 @@ Other integrative resources, some still under development, include [Wikidata](ht
 
 ### Rephetio
 
-
+Project Rephetio is the name of the [study](https://git.dhimmel.com/rephetio-manuscript/) that created Hetionet and applied it repurpose drugs [@rephetio].
```

Comment:
It feels like it'd be helpful to discuss things a bit more broadly - for example, supervised ML being applied to these types of graphs. Is there other work that touches on these ideas with similar approaches?

Reply:
definitely. this is an appropriate section to discuss related approaches. I collected some references in https://github.com/greenelab/connectivity-search-manuscript/pull/6/commits/2244120e91bd4781bf40781c37155403ec6082e0. Will merge this PR now and expand the other works stub subsequently.